org/mozilla/jss/util/NativeProxy.java

```diff
@@ -68,8 +77,10 @@ public boolean equals(Object obj) {
      * Subclasses of NativeProxy must define this method to clean up
      * data structures in C code that are referenced by this proxy.
      * releaseNativeResources() will usually be implemented as a native method.
-     * <p>You don't call this method; NativeProxy.finalize() calls it for you.
-     * <p>You must declare a finalize() method which calls super.finalize().
+     * <p>You don't call this method; NativeProxy.finalize() calls it for you.</p>
+     * <p>You must declare a finalize() method which calls super.finalize().</p>
```

Comment:
Should we change this doc to call close() instead? Existing code will still call finalize(), but new code should call close() or use AutoClosable.

Reply:
Is this really correct anywho? I'm kinda under the impression that it isn't strictly necessary. 

 - If you're not freeing / clearing / &c. anything else, I don't see the point in overriding finalize(); won't the child's freeNativeResources(...) be called since NativeProxy doesn't define one?
 - If you are freeing something else, then it is obvious you need to override close(...) / finalize(...).

Thoughts?

Reply:
As discussed on IRC we can drop the comment.